addons/web_editor/static/src/components/media_dialog/media_dialog.js

```diff
@@ -144,7 +144,12 @@ export class MediaDialog extends Component {
 
     async save() {
         const selectedMedia = this.selectedMedia[this.state.activeTab];
-        if (selectedMedia.length) {
+        // TODO In master: clean the save method so it performs the specific
+        // adaptation before saving from the active media selector and find a
+        // way to simply close the dialog if the media element remains the same.
```

Comment:
@xO-Tx I would have thought you would have adapted this directly here... Since this one is merged, can you create a task for it?